### PR TITLE
Added bump command

### PR DESCRIPTION
**Pull request by poppin-fresh**
**Please describe the changes this PR makes and why it should be merged:**

Moves a track from n index in the queue to the front and shifts the array
e.g. (>bump 12) Will move song from position 12 to 1

**Status and versioning classification:**

### CODE REVIEW

**Comment by Kakarot-0690** [comment]
have you tested this properly?

**Comment by poppin-fresh** [comment]
> have you tested this properly?

Yes.

**Comment by Kakarot-0690** [comment]
> > have you tested this properly?
> 
> Yes, extensively. There shouldn't be any problems, however if you find something please let me know.

yup looks pretty good to me

File: commands/bump.js [hunk 1]
```diff
@@ -21,15 +21,17 @@ module.exports = {
...
+        }
         
-        // Swap array elements
-        [player.queue[0], player.queue[trackNum]] = [player.queue[trackNum], player.queue[0]];
-		client.sendTime(message.channel, "✅ | **" + player.queue[0].title + "** has been moved to the front of the queue.");
+        // Remove from and shift array
+        const track = player.queue[trackNum];
+        player.queue.splice(trackNum, 1);
+        player.queue.unshift(track);
+		client.sendTime(message.channel, "✅ | **" + track.title + "** has been moved to the front of the queue.");
```
Comment (DarrenOfficial) [inline commands/bump.js:34]: /root/sudhan/poppin-fresh/commands/bump.js:34
                client.sendTime(message.channel, "✅ | **" + track.title + "** has been moved to the front of the queue.");
                                                                  ^
TypeError: Cannot read property 'title' of undefined
    at Object.run (/root/sudhan/poppin-fresh/commands/bump.js:34:53)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)

**Review by DarrenOfficial — CHANGES_REQUESTED** [review]
Running Bump without number crashed the bot, please re-check
```bash
/root/sudhan/poppin-fresh/commands/bump.js:34
                client.sendTime(message.channel, "✅ | **" + track.title + "** has been moved to the front of the queue.");
                                                                  ^

TypeError: Cannot read property 'title' of undefined
    at Object.run (/root/sudhan/poppin-fresh/commands/bump.js:34:53)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

**Comment by poppin-fresh** [comment]
@DarrenOfficial fixed in cc04f2f.
Had this in my version but pushed an older one, my bad.

**Comment by Kakarot-0690** [comment]
> @DarrenOfficial fixed in [cc04f2f](https://github.com/SudhanPlayz/Discord-MusicBot/commit/cc04f2fcbd189a343522deefadc11efca2978839). Had this in my version but pushed an older one, my bad.

tested and tried, works

**Pull request merged**